website/guides/workflow.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Workflow
+description: How does gql.tada and its LSP fit into your wofklow?
+---
+
+# Workflow
+
+There are some general workflows that are distinct locally (i.e. the LSP can take care of it)
+vs remotely. In this guide we'll look at how we can get local/remote to being consistent.
+
+> [!NOTE]
+>
+> This assumes that every `gql.tada` file is checked into VC.
```

Comment:
Hm, I'm not sure what the note is getting at, it doesn't seem to prompt or warn the user of anything, and seems a little hard to read. (VC is also not a common abbreviation over "checked into the repository")

Should we remove this and make case-by-case evaluations of what the trade-offs are of checking files in or not later on?